react/components/SpecificationsTranslation/SpecificationsForm.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, SyntheticEvent } from 'react'
+import { Input, Textarea, Button } from 'vtex.styleguide'
+import { useMutation } from 'react-apollo'
+
+import { useLocaleSelector } from '../LocaleSelector'
+import translateSpecificationMutation from '../../graphql/translateSpecification.gql'
+import { useAlert } from '../../providers/AlertProvider'
+import useFormTranslation from '../../hooks/useFormTranslation'
+import ActionButtons from '../ActionButtons'
+
+interface SpecificationsFormProps {
+  specificationInfo: FieldInputTranslation
+  specificationId: string
+  updateMemoSpecifications: (
+    value: React.SetStateAction<{
+      [Identifier: string]: SpecificationsData
+    }>
+  ) => void
+}
+
+const SpecificationsForm: FC<SpecificationsFormProps> = ({
+  specificationInfo,
+  specificationId,
+  updateMemoSpecifications,
+}) => {
+  const {
+    formState,
+    canEdit,
+    handleInputChange,
+    changed,
+    handleToggleEdit,
+  } = useFormTranslation(specificationInfo)
+
+  const { isXVtexTenant, selectedLocale } = useLocaleSelector()
+  const [translateSpecification, { loading }] = useMutation<
+    { translateSpecification: boolean },
+    { args: Specifications; locale: string }
+  >(translateSpecificationMutation)
+
+  const { openAlert } = useAlert()
+  const handleSubmit = async (e: SyntheticEvent) => {
+    e.preventDefault()
+    if (loading) {
+      return
+    }
+    const args = {
+      ...formState,
+      ...{ fieldId: specificationId },
+    }
+    try {
+      const { data, errors } = await translateSpecification({
+        variables: {
+          locale: selectedLocale,
+          args,
+        },
+      })
+      const { translateSpecification: translateSpecificationResult } =
+        data ?? {}
+      if (translateSpecificationResult) {
+        openAlert('success', 'Specifications')
+      }
+      if (errors?.length) {
+        throw new TypeError('Error translation Specifications')
+      }
+    } catch (err) {
+      openAlert('error', 'Specifications')
+    }
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div className="mb5">
+          <Input
+            label="Name"
+            value={formState.name}
+            name="name"
+            disabled={isXVtexTenant || !canEdit}
+            onChange={handleInputChange}
+          />
+        </div>
+        <div className="mb5">
+          <Textarea
+            resize="none"
+            label="Description"
+            value={formState.description}
+            name="description"
+            disabled={isXVtexTenant || !canEdit}
+            onChange={handleInputChange}
+            required
+          />
+        </div>
+        <div className="mb5">
+          <Input
+            label="Field type"
+            value={formState.fieldTypeName}
+            name="fieldTypeName"
+            disabled={isXVtexTenant || !canEdit}
+            onChange={handleInputChange}
+          />
+        </div>
```

Comment:
@acapeca we shouldn't be displaying the Field Type value. 
this is going to be used internally for us to identify which Specifications are of type text from the rest. 
please take it out.

Reply:
Done in latest commit